convert_to_multi_instance.sh

```diff
@@ -53,6 +60,205 @@ function prepare_cluster() {
     # uninstall singleton services
 }
 
+function collect_data() {
+    title "collecting data"
+    msg "-----------------------------------------------------------------------"
+
+    cs_operator_namespace=$(${OC} get pod --all-namespaces | grep operand-deployment-lifecycle-manager | awk '{print $1}')
+    echo MasterNS:${cs_operator_namespace}
+    cs_operator_channel=$(${OC} get sub ibm-common-service-operator -n ${cs_operator_namespace} -o yaml | yq ".spec.channel") 
+    echo channel:${cs_operator_channel}   
+    catalog_source=$(${OC} get sub ibm-common-service-operator -n ${cs_operator_namespace} -o yaml | yq ".spec.source")
+    echo catalog_source:${catalog_source}   
+    
+}
+
+# delete all CS pod and read configmap
+function restart_CS_pods() {
+    title "restarting ibm-common-service-operator pod"
+    msg "-----------------------------------------------------------------------"
+    ${OC} get pod --all-namespaces | grep ibm-common-service-operator | while read -r line; do
+        local namespace=$(echo $line | awk '{print $1}')
+        local cs_pod=$(echo $line | awk '{print $2}')
+
+        msg "deleting pod ${cs_pod} in namespace ${namespace} "
+        ${OC} delete pod ${cs_pod} -n ${namespace} 
+        if [[ $? -ne 0 ]]; then
```

Comment:
this if will actually never be run if there is error because of this line at the top of the script `set -o errexit`. This sets the option for the script to exit immediately if there is any error. So you will actually need to use that logical or `||` I commented in line 85.